### PR TITLE
Issue118

### PR DESCRIPTION
**Pull request by yunfeiguo**
- [ ] #118 
- [ ] Blocking PRs: ADD HERE IF ANY
* People
  - [ ] Reviewers (>=1): ADD HERE
  - [ ] Merger (>=1): ADD HERE
  - [ ] FYI: ADD HERE IF ANY
- [ ] Release note (if needed)
- [ ] No code copied from outside Bina 
- [x] Tests (Unit, Workflow, ITL & Gherkin) are passing
- [ ] PR labels, milestones & assignees

# Summary

* correct `CN` for DGV-converted VCF
* make default `CN=2` for `DUP` variants (earlier it was 1, which is same as reference).

### CODE REVIEW

File: src/main/java/com/bina/varsim/util/DGVparser.java [hunk 1]
```diff
@@ -146,7 +146,11 @@ public Variant parseLine() {
         VariantType type = getVariantType(ll[5].toLowerCase());
 
         // TODO right now we treat all gains as tandem duplications
-        int observedgains = (ll[15].length() > 0) ? Integer.parseInt(ll[15]) : 0;
+	if (ll[15].length() == 0)
+	    throw new RuntimeException("no observedgain");
+        int observedgains = Integer.parseInt(ll[15]);
+	if (observedgains == 0)
+	    throw new RuntimeException("observedgains = 0, not a variant, skipping");
```
Comment (marghoob) [inline src/main/java/com/bina/varsim/util/DGVparser.java]: it's not really "skipping" if you throw an exception. Is this exception being caught somewhere?

Comment (yunfeiguo) [reply src/main/java/com/bina/varsim/util/DGVparser.java]: yes, see https://github.com/bioinform/varsim/pull/138/files/542467fe7d997ed140f533603dec054c13858d7f#diff-aeacb89a2526bc17574fdcc1e9e11eb0L213

**Comment by yunfeiguo** [comment]
@marghoob is this okay for merging?

**Comment by marghoob** [comment]
👍

**Pull request merged**